arcgis-ios-sdk-samples/Augmented reality/Collect data in AR/README.md

```diff
@@ -0,0 +1,62 @@
+
+## Use case
+
+You can use AR to quickly photograph an object and automatically determine the object's real-world location, facilitating a more efficient data collection workflow. For example, you could quickly catalog trees in a park, while maintaining visual context of which trees have been recorded - no need for spray paint or tape.
+
+## How to use the sample
+
+Before you start, go through the on-screen calibration process to ensure accurate positioning of recorded features. Feature points detected by ARKit are shown to help you understand what the application sees and where you can tap to collect accurate features.
+
+When you tap, an orange diamond will appear at the tapped location. You can move around to visually verify that the tapped point is in the correct physical location. When you're satisfied, tap the '+' button to record the feature. An image from the camera feed will automatically be attached to the recorded feature.
+
+> ⚠️ **WARNING**: collection of photos is completely automatic; consider your surroundings when adding features.
+
+## How it works
+
+1. Create an `ArcGISARView`. Set the atmosphere effect to `none` and the space effect to `transparent`. Create and show a scene in the scene view.
+2. Load the feature service and display it with a feature layer.
+3. Create and add the elevation surface to the scene.
+4. Create a graphics overlay for planning the location of features to add. Configure the graphics overlay with a renderer and add the graphics overlay to the scene view.
+5. When the user taps the screen, use `arcGISARView.arScreenToLocation` to find the real-world location of the tapped object using ARKit/ARCore plane detection.
+6. Add a graphic to the graphics overlay preview where the feature will be placed and allow the user to visually verify the placement.
+7. When the user presses the button, take the current AR frame from the `arView.arSCNView.session`. Rotate the image appropriately and convert it to a JPEG for efficient storage.
+8. Prompt the user for a tree health value, then create the feature. Upon successful creation of the feature, use `feature.addAttachment` to add the image.
+
+## Relevant API
+
+* ArcGISARView
+* GraphicsOverlay
+* SceneView
+* Surface
+
+## About the data
+
+The sample uses a publicly-editable sample [tree survey feature service](https://arcgisruntime.maps.arcgis.com/home/item.html?id=8feb9ea6a27f48b58b3faf04e0e303ed). You can use AR to quickly record the location and health of a tree while seamlessly capturing a photo.
+
+## Additional information
+
+This sample requires a device that is compatible with ARKit 1 on iOS or ARCore 1.8 on Android.
+
+There are two main approaches for identifying the physical location of tapped point:
+
+* **ArcGISARView.arScreenToLocation** - uses plane detection provided by ARKit/ARCore to determine where _in the real world_ the tapped point is.
```

Comment:
I recommend removing the 'or ARCore' parts since in this context it is iOS specific. Applies to the other readmes as well.

Reply:
Also affects line 58 below.